Cilsil/Log.cs

```diff
@@ -125,20 +159,42 @@ public static void Write(string s, ConsoleColor c)
         /// </summary>
         public static void WriteError(string s) => WriteLine(s, ConsoleColor.Red);
 
+        /// <summary>
+        /// TODO: use https://nlog-project.org or log4net instead of this class.
+        /// </summary>
+        public static void DebugWriteError(string s)
+        {
+            if (logLevel == LogLevel.Debug)
+            {
+                WriteLine(s, ConsoleColor.Red);
```

Comment:
Maybe instead of calling WriteLine directly, I could call the original WriteError method?